cms/envs/bok_choy.env.json

```diff
@@ -74,7 +74,6 @@
         "SUBDOMAIN_BRANDING": false,
         "SUBDOMAIN_COURSE_LISTINGS": false,
         "ALLOW_ALL_ADVANCED_COMPONENTS": true,
-        "ALLOW_COURSE_RERUNS": true,
```

Comment:
@cahrens Yes, but it didn't work for some reason. You can see my changes here:

https://github.com/edx/edx-platform/pull/6944